lib/LaTeXML/Core/Parameters.pm

```diff
@@ -60,7 +60,7 @@ sub readArguments {
   my ($self, $gullet, $fordefn) = @_;
   my @args = ();
   foreach my $parameter (@$self) {
-    my $value = $parameter->read($gullet);
+    my $value = $parameter->read($gullet, $fordefn);
```

Comment:
strange oversight, adding fordefn back in makes the parameter read errors much saner (as opposed to `wrong argument of undef`)